### DISCUSSION TITLE
这个规则只能使用TikTok 21.1.0 版本吗

### DISCUSSION THREAD

**Discussion by LittleXingzzZ** [Semporia/TikTok-Unlock#273] [Q&A]
最新版本26.3、26.2可行吗

**Comment by Ledrong** [comment]
你好，我是來自俄羅斯的學生。我想在https://buff.163.com/平台進行交易。但為此我需要驗證我的帳戶，而這只能使用支付寶和steam來完成。請您只需要驗證一下，一切都是安全的。我只需要一個個人資料
